gui/src/renderer/components/SearchBar.tsx

```diff
@@ -67,6 +67,7 @@ export const StyledClearIcon = styled(ImageView)({
 });
 
 interface ISearchBarProps {
+  searchInputRef?: React.Ref<HTMLInputElement>;
```

Comment:
In your [previous suggestion](https://github.com/mullvad/mullvadvpn-app/pull/6832#issuecomment-2371063363) @raksooo , you showed me `AutoSizingTextInputWithRef` as an example:

https://github.com/mullvad/mullvadvpn-app/blob/d35866d337647a26dc8ffc2e875212306dd7c0ce/gui/src/renderer/components/cell/Input.tsx#L196

This components consumes a `forwardedRef` parameter, the `ref` object does not come from its `props`.
But here I was only able to pass it via a prop.

Reply:
To use the `ref`-prop you will need to wrap the component in `React.forwardRef`.
```
function SearchBar(props: ISearchBarProps, forwardedRef: React.Ref<HTMLInputElement>) { ... }

export default React.forwardRef(SearchBar);
```

Then you won't have to include it in props and you'll be able to use `SearchBar` like this:
```
<SearchBar ref={searchInputRef} ... />
```

I think `React.forwardRef` won't be needed in future versions of react, but for now we'll need to add it :)